Tasks/SshV0/ssh.ts

```diff
@@ -63,10 +63,10 @@ async function run() {
                 .filter(s => s.length > 0);
         } else if (runOptions === 'inline') {
             let inlineScript: string = tl.getInput('inline', true);
-            if (inlineScript && !inlineScript.startsWith('#!')) {
-                const bashHeader: string = '#!/bin/bash';
-                tl.debug('No script header detected.  Adding: ' + bashHeader);
-                inlineScript = `${bashHeader}${os.EOL}${inlineScript}`;
+            const interpreterCommand: string = tl.getInput('interpreterCommand');
+            if (inlineScript && !inlineScript.startsWith('#!') && interpreterCommand) {
+                tl.debug('No script header detected.  Adding: #!' + interpreterCommand);
```

Comment:
For windows remote host we just have to use an empty string as the field value.This will prevent adding interpreter command to the beginning of the script.